src/guide/configure/working-with-the-keys.md

```diff
@@ -0,0 +1,78 @@
+# Working with the cryptographic keys
+
+## Creating and using the keys
+
+::: warning
+We have several code tutorials and an article about [public-key cryptography](./keys.html). Despite us using `JSON` in tutorial examples to store the keys plainly, this approach is only used for demonstration purposes because you need a more secure one for production use. Iroha 2 will directly support better practices with time.
```

Comment:
```suggestion
We have created several code tutorials and an article about [public-key cryptography](./keys.html). Despite us using `JSON` in tutorial examples to store the keys plainly, this approach is only used for demonstration purposes because you need a more secure one for production use. Iroha 2 will directly support better practices with time.
```